README.md

```diff
@@ -27,9 +30,17 @@ API Definition
 --------------
 
 See the human-readable [Reference Documentation](https://ga4gh.github.io/task-execution-schemas/docs/)
-and the [OpenAPI YAML description](openapi/task_execution_service.openapi.yaml). You can also explore the specification in the [Swagger Editor](https://editor.swagger.io/?url=https://ga4gh.github.io/task-execution-schemas/openapi.yaml).
+You can also explore the specification in
+the [Swagger Editor](https://editor.swagger.io/?url=https://ga4gh.github.io/task-execution-schemas/openapi.yaml)**
+*Manually load the JSON if working from a non-develop branch version.* Preview documentation from
+the [gh-openapi-docs](https://github.com/ga4gh/gh-openapi-docs) for the development
+branch [here](https://ga4gh.github.io/task-execution-schemas/preview/develop/docs/index.html)
+
+> All documentation and pages hosted at 'ga4gh.github.io/task-execution-schemas' reflect the latest API release from
+> the `master` branch. To monitor the latest development work, add 'preview/\<branch\>' to the URLs above (e.g., '
+> ga4gh.github.io/task-execution-schemas/preview/\<branch\>/docs'). To view the latest *stable*
+> development API specification, refer to the `develop` branch.
```

Comment:
Are we sure this is still accurate with the new build pipeline?

Reply:
Yep that is true. Double checked it https://ga4gh.github.io/task-execution-schemas/preview/develop/docs/

Reply:
Thanks. So we can close this convo, right? If so, please go ahead.